_layouts/job_post.html

```diff
@@ -0,0 +1,6 @@
+---
+layout: default
+title_prefix: "Compiler Jobs:"
+stylesheet: "/styles/jobs.css"
```

Comment:
This also seems slightly problematic to have (to know) to use the `/styles` prefix here, vs. having the template figure out where the file is based on which one you want.